.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   },
   plugins: ["@typescript-eslint", "cypress"],
   rules: {
+    "indent": "off",
```

Comment:
please do not remove the indent rule

Reply:
I am replacing the es lint ident rule with prettier

Reply:
then adjust prettier to have the right tab width

Reply:
Prettier is opinionated and has its own way of format the code and eslint demands another one. I'm just overwriting the ident rule, I can't adjust prettier because it is opinionated and to avoid the conflict of both I prefer to leave prettier for ident and eslint for the rest.

If you want to know more about prettier, check this https://prettier.io/docs/en/index.html there are very few rules that can be customized, I tried to format this but it was impossible.

Reply:
I'm start hating prettier, also had same issue, so it seems that is not a good choice, as eslint follows google standards.